src/node_options.h

```diff
@@ -176,8 +176,8 @@ class EnvironmentOptions : public Options {
 #endif  // HAVE_INSPECTOR
   std::string redirect_warnings;
   std::string diagnostic_dir;
-  std::string env_file;
-  std::string optional_env_file;
+  std::vector<std::string> env_files;
+  std::vector<std::string> optional_env_files;
```

Comment:
I don't have the C++ knowledge to suggest actual code changes, but what about using only one vector, that contains a struct with the string and whether it's optional or not?

Reply:
I'm not sure that's possible. AFAIK The options parser parses each flag to its own value (in this case a vector). I'm not sure you can map multiple flags to the same vector.

Reply:
I'm sure you can do it if you change something in the parser's code.